src/containers/DataPackageView.js

```diff
@@ -18,8 +19,12 @@ export class DataPackageViewContainer extends React.Component {
   }
 
   componentDidMount(){
-    let url = "https://raw.githubusercontent.com/anuveyatsu/test_data/master/datapackage.json";
+    let baseUrl = "https://bits.staging.datapackaged.com/metadata/";
```

Comment:
This will be passed from flask side. Right now this is hand coded as backend api does not this field.